### PR TITLE
July 2018 Slides

### PR DESCRIPTION
**Pull request by jessedc**
I copy pasted nearly everything from June.

### CODE REVIEW

File: intro-2018-07.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+- realestate.com.au, Beanhunter, RMIT
+- Melbourne CocoaHeads organiser since ~2014
+
+---
+
+# Upcoming Events
+
+![left](assets/highres_469612543.jpeg)
+
+- Hack Night - July  18th @ Cognizant
```
Comment (bok-) [inline intro-2018-07.md]: Double space

File: intro-2018-07.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+# Tonight's Agenda
+
+Time|Speaker|Topic|
+---|---|---
+6:00 - 6:30 | Arrival and :pizza:
+6:30 - 6:40 | Intro & Who's Hiring
+6:45 - 7:15 | Vlas Voloshin | Marzipan's Internals
+7:15 - 7:45 | Prasanna Gopalakrishnan | Architecture for Scaling Mobile
+
```
Comment (bok-) [inline intro-2018-07.md]: Still might be good to have a pub line in here

**Review by bok- — APPROVED** [review]

File: intro-2018-07.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+---
+
+# Upcoming Events
+
+![left](assets/highres_469612543.jpeg)
+
+- Hack Night - July 18th @ Cognizant
+- Drinks Night - July 31st
+- NS Breakfast - August 3rd
```
Comment (cuteiosdev) [inline intro-2018-07.md:53]: DevWorld? Might be good to mention it.

Comment (bok-) [reply intro-2018-07.md:53]: I have a section in the AWS group intro now about related conferences and groups. Its gotten good feedback so far

**Pull request merged**